### ISSUE TITLE
New `auto_merge` option logic not in tagged release

### ISSUE THREAD

**Issue by Ignigena** [chrnorm/deployment-action#4]
Your action yaml is pointing to `lib/main.js` but the new `auto_merge` option is not present here on the tagged release: https://github.com/chrnorm/deployment-action/blob/v1.1.0/lib/main.js

This is causing the action to fail for us with similar errors to #1 

```
Run chrnorm/deployment-action@v1.1.0
  with:
    environment: production
    initial_status: success
    target_url: ***
    token: ***

##[error]HttpError: Conflict merging development into refs/heads/master.
##[error]Conflict merging development into refs/heads/master.
##[error]Node run failed with exit code 1
```

**Comment by chrnorm** [comment]
Hi @Ignigena, thanks for the issue. I was wondering why I was still having similar errors to you myself.. It turns out that while I added the `auto_merge` option to the TypeScript code, I forgot to recompile the JavaScript `main.js` file for the release.

I've just released a [new version](https://github.com/chrnorm/deployment-action/releases/tag/v1.1.1) which fixes this - `chrnorm/deployment-action@v1.1.1`. Please try that and let me know if that works.

**Comment by Ignigena** [comment]
@chrnorm That did the trick! Thanks so much for moving so quick 👍
